extensions/dacpac/src/test/dacpac.test.ts

```diff
@@ -7,8 +7,9 @@
 import 'mocha';
 import * as should from 'should';
 import * as os from 'os';
-import { isValidFilenameCharacter, sanitizeStringForFilename } from '../wizard/api/utils';
+import { isValidFilenameCharacter, sanitizeStringForFilename, isValidBasename } from '../wizard/api/utils';
 
+const isWindows = os.type().includes('Windows');
```

Comment:
Any reason you changed this? Typically we've been doing the platform==='win32' check as you had before

Reply:
it's what I had in another file, but I changed both of them to be platform==='win32' now